using-mobile-extensions/adobe-edge-identity/edge-identity-faq.md

```diff
@@ -125,4 +125,184 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 }
 ```
 {% endtab %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+
+## Q: Will an existing Experience Cloud ID (ECID) migrate to the AEP Edge Identity extension?
```

Comment:
Overall what do you think about renaming Edge Identity -> Identity for Edge Network and Identity direct -> Identity for ECID Service